src/modules/logger/logger.cpp

```diff
@@ -476,6 +476,7 @@ bool Logger::copy_if_updated_multi(LoggerSubscription &sub, int multi_instance,
 void Logger::add_default_topics()
 {
 	add_topic("vehicle_attitude", 10);
+	add_topic("vehicle_attitude_groundtruth", 10);
```

Comment:
Should we add an ifdef here for SITL? Because the logger needs to iterate all subscribed topics on each interval, and checking for a topic that is never published just wastes CPU cycles.